airflow/api/client/local_client.py

```diff
@@ -36,18 +38,20 @@ def delete_dag(self, dag_id):
         return f"Removed {count} record(s)"
 
     def get_pool(self, name):
-        the_pool = pool.get_pool(name=name)
+        the_pool = Pool.get_pool(pool_name=name)
```

Comment:
Since we’re no longer importing a module named `pool`, let’s change this variable to `pool` which is much prettier than `the_pool`.